src/__tests__/stateToHTML-test.js

```diff
@@ -27,7 +27,8 @@ describe('stateToHTML', () => {
     let {description, state, html} = testCase;
     it(`should render ${description}`, () => {
       let contentState = convertFromRaw(state);
-      expect(stateToHTML(contentState)).toBe(html);
+      let options = {customStyleMap: {'RED': { color: 'red' }}};
```

Comment:
Nit: Inconsistent spacing. Also, no need to quote object key RED. (ESLint didn't catch this?)